tradeexecutor/strategy/pandas_trader/runner.py

```diff
@@ -270,8 +270,8 @@ def report_strategy_thinking(self,
                 
             logger.trade(buf.getvalue())
 
-            large_image = self.run_state.visualisation.large_image
-            post_logging_discord_image(large_image)
+            # there is already a warning in refresh_visualisations for pair count > 3
```

Comment:
Can you add a log message "cannot post image, too many N pairs" or similar so that we know it tried to post the image but could not do it